docs/sdkx-cocos2dx/users.mdx

```diff
@@ -0,0 +1,294 @@
+description: Learn more about identifying and managing users.
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+<Admonition type="info" title="Note">
+
+All the public APIs in the SDK should be called after initializing the SDK via [install API](/sdkx-cocos2dx/getting-started#initializing)
+
+</Admonition>
+
+## How to Identify and Manage Users {#identify-and-manage-users}
+
+### Logging in users {#logging-in-users}
+
+#### Overview
+
+A logged-in user is someone who can contact your support team only after providing a username and password. If you have logged-in users, we highly recommend passing the user's identifiers (user ID and/or email) using the Login API so that your Agents can provide a personalized support experience to your users. Using the Login API also ensures that a user's conversations are available only to them when they log in.
+
+<Admonition type="info" title="Note">
+
+User's conversation will only be available on the device used to start the conversation. The conversation will not be available across devices.
+
+</Admonition>
+
+#### What to provide as identifiers
+
+You can provide either `userID` and/or `userEmail` to identify your users. We highly recommend using a user ID to identify your users. However, if you use emails to identify your users, then you should pass them in the `userEmail` field as a map in the `HelpshiftCocos2dx::login(cocos2d::ValueMap userData)` API. The following logic applies when you use both `userID` as well as `userEmail`:
+
+- When looking up users, the priority of `userId` is higher than that of `userEmail`
+- When the user ID matches an exisiting user, that user's email will be updated (if the email is provided)
+- When the email matches an existing user, the following logic applies:
+  - If the user ID does not exist for a user matched by email, then the user ID will be added for that user (if a user ID is provided)
+  - If the user ID already exists on the user matched by email, then a new user will be created (if a different user ID is provided)
+
+#### How to use
+
+You should call Helpshift SDK's Login API whenever the user successfully logs into your app. The Login API takes the following parameters:
+
+<table>
+  <tr>
+    <th>Parameter</th>
+    <th>Require/Optional</th>
+    <th>SDK Description</th>
+    <th>Inportant Considerations</th>
+  </tr>
+  <tr>
+    <td>userName</td>
+    <td>Optional</td>
+    <td>Provide the name you'd like your Agents to use when interacting with end users. If you don't know the user's name, you can leave it blank, and the Identity Bot (when enabled) will ask that user for their name. If you provide a value, the Identity Bot will not ask the user for their name again.</td>
+    <td>Max length: 255 characters. Values larger than this will be truncated.</td>
```

Comment:
we have released an update in this table recently.
Please refer to other plugins doc and update the entire content here accordingly.